store/errors.go

```diff
@@ -0,0 +1,25 @@
+package store
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	DefaultCodespace sdk.CodespaceType = "store"
+
+	CodeOutputDNE   sdk.CodeType = 1
+	CodeOutputSpent sdk.CodeType = 2
+	CodeAccountDNE  sdk.CodeType = 3
+)
+
+func ErrOutputDNE(codespace sdk.CodespaceType, msg string, args ...interface{}) sdk.Error {
```

Comment:
codespace isn't needed with these wrappers. These errors should only arise within this module